### PR TITLE
fix: update balsamic tags

### PR DESCRIPTION
**Pull request by ivadym**
## Description

Update tags according to `BALSAMIC 8.x.x`.

### How to prepare for test
- [ ] ssh to hasta.scilifelab.se
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-hermes-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

**Comment by ivadym** [comment]
- Update tags for WGS TUMOR_ONLY CASE.
- Update tags for WGS TUMOR_NORMAL CASE.

Screen Shot 2021-11-16 at 16 41 01

**Comment by ivadym** [comment]
Updated tags for `PANEL TUMOR_ONLY`, `PANEL TUMOR_NORMAL`, `PANEL UMI TUMOR_NORMAL` and `PANEL UMI TUMOR_ONLY` cases.

The tags `{"tmb", "stat-somatic-tmb", "vardict"}` have been removed as the balsamic rule (`vep_stat`) that was generating them no longer exists.

Tested on `hasta`:
Screen Shot 2021-11-17 at 15 53 25

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
+## [1.5.5]
+### Added
```
Comment (ashwini06) [inline CHANGELOG.md:8]:
```suggestion
### Changed
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+- Update tags according to new BALSAMIC features
```
Comment (ashwini06) [inline CHANGELOG.md:9]:
```suggestion
- Removed cnv tags from balsamic common tags
- Added cnv tags for balsamic panel (tumor-only and tumor-normal)
- Removed vardic tmb stat tags
```

**Review by ashwini06 — CHANGES_REQUESTED** [review]
Look good 👍🏼

**Review by ashwini06 — APPROVED** [review]

**Review by Mropat — APPROVED** [review]

**Pull request merged**